public/entity2/entitysystem.h

```diff
@@ -22,6 +22,12 @@ class IEntityResourceManifestBuilder;
 class ISpawnGroupEntityFilter;
 class IHandleEntity;
 
+class CEntitySystem;
+class CGameEntitySystem;
+
+extern CEntitySystem* EntitySystem();
+extern CGameEntitySystem* GameEntitySystem();
```

Comment:
Do we need the distinction between the two? Since they would anyway return the same pointer for both on practice, so to not duplicate code I guess we can just use ``GameEntitySystem`` everywhere instead, what do you think?

Reply:
As far as I've seen, the game uses 2 pointers so I made a distinction. Yes, I think that it would be unnecessary to separate them, and we can only use `GameEntitySystem()` in our case.